examples/src/app/layout.tsx

```diff
@@ -11,11 +11,7 @@ export default function RootLayout({ children }: { children: React.ReactNode })
   return (
     <html lang="en">
       <body className="bg-[radial-gradient(at_bottom_left,_var(--tw-gradient-stops))] from-pink-50 to-pink-900">
-        <Sidebar>
-          <div className="absolute w-full h-full top-0 pt-14 lg:relative lg:w-auto lg:pt-0 bg-[radial-gradient(at_bottom_left,_var(--tw-gradient-stops))] from-white/90 via-pink-50/90 to-pink-100/90 sm:mr-2 sm:rounded-lg">
```

Comment:
Since we moved this wrapper into the `(demos)` directory, we lost the styles from the TLDraw demo which sits outside `(demos)`.

To fix this, can we add `sm:mr-2 sm:rounded-lg overflow-hidden bg-white` to the `tldraw__editor` div in the `TldrawExample` component in `Tldraw.tsx`?